### DISCUSSION TITLE
Integrate AdGuard to IOTstack Project

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#2874] [Q&A] [answered]
[IOTstack](https://github.com/SensorsIot/IOTstack) is a easy way to setup and run IoT stacks on Raspberry PI, they already support many famous docker friendly projects including Pi-Hole, I think it would be good to support AdGuardHome as well.
I tried and asked them to help to build support for AH but there are problems: https://github.com/SensorsIot/IOTstack/issues/297

Could you please review the issue and give advice how to solve it?

Regards,
Erica

**Comment by ameshkov** [comment, marked as answer]
As I see most of the confusion comes from the fact that AdGuard Home prints internal docker addresses in the UI, but this is kinda normal - it simply cannot know the host machine addresses. And with published ports, AGH should be accessible on the outside, i.e. on `host_ip:published_port`.

I assume that anyone using docker is familiar with this behavior. I guess the only thing that we can is improve the documentation a little bit and explain this.
